core/src/main/scala/com/nvidia/spark/rapids/tool/planparser/DataWritingCommandExecParser.scala

```diff
@@ -105,14 +110,20 @@ object DataWritingCommandExecParser {
     // if overwriteByExprExecV1 is not deltaLakeProvider, then we want to mark it as unsupported
     overwriteByExprExecV1 -> "unknown",
     // if atomicReplaceTableExec is not deltaLakeProvider, then we want to mark it as unsupported
-    atomicReplaceTableExec -> "unknown"
+    atomicReplaceTableExec -> "unknown",
+    // if atomicCreateTableExec is not deltaLakeProvider, then we want to mark it as unsupported
+    atomicCreateTableExec -> "unknown"
   )
 
   // Checks whether a node is a write CMD Exec
   def isWritingCmdExec(nodeName: String): Boolean = {
     logicalWriteCommands.exists(nodeName.contains(_)) || DeltaLakeHelper.accepts(nodeName)
   }
 
+  def getPhysicalExecName(opName: String): String = {
```

Comment:
Does this look risky? Earlier if the logical operator was not present in `logicalToPhysicalCmdMap` it would throw an error. Now we would be returning `defaultPhysicalCMD` value.

Reply:
It is safer to have a default value. For example, this allows us to use the same  method to get a physical representation of Delta Operators that are missing in the CSV files (i.e., `MergeIntoCommandEdge`, `WriteIntoDeltaCommand`, and `SaveIntoDataSrcCMD`).
Otherwise, we will have fill in the content of the map with all the Operators in Hive/Delta, which can be crash-able if someone forget to update the map with the new entry.

Reply:
Okay. This is definitely reducing redundancy. We should be aware of this default behaviour.